cmd/wstail/tail.go

```diff
@@ -273,6 +275,26 @@ func serveWs(w http.ResponseWriter, r *http.Request) (err error) {
 	return nil
 }
 
+// downloadLogs serves /v1/download
+//
+// it compresses the file with aggregated logs and pushes it to the response
+// stream
+func downloadLogs(w http.ResponseWriter, r *http.Request) error {
+	file, err := os.Open(filePath)
```

Comment:
I guess we could include older logs too. Would they be named just like `filePath+.<number>` (e.g. `/var/log/messages.1, /var/log/messages.2`) or they are named differently?